collectors/iscsi.py

```diff
@@ -92,6 +97,17 @@ class ISCSIGateway(BaseCollector):
     def __init__(self, *args, **kwargs):
         BaseCollector.__init__(self, *args, **kwargs)
 
+        # Since the module can be imported by a parent class but not
+        # instantiated, the rtslib import is deferred until the first instance
+        # of the the class is created. This keeps the parent module simple
+        # and more importantly generic
+        if 'rtslib_fb.root' not in sys.modules.keys():
+
+            try:
+                import rtslib_fb.root as RTSRoot
+            except ImportError:
+                raise ImportError("rtslib_fb python package is missing")
```

Comment:
I don't like conditional imports at the best of times, but it's necessary to keep the collectd plugin as generic as possible. With an import like this I'd always want to try the import and catch the error to show the logic if nothing else....just in case the runtime environment doesn't have rtslib (that would be weird but possible)